tests/functional/remote/02-file-install-configured.t

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#-------------------------------------------------------------------------------
+# Checks default files (app, bin, etc, lib) are correctly installed on the
+# remote platform.
```

Comment:
Could this functional test be combined with the previous one (to slow down the proliferation of slow function tests a little)?  They seem to be almost identical (default includes vs explicit ones).

Reply:
Always happy to reduce the proliferation of slow functional tests! I have now combined them to one test, although I have kept it as two workflow runs, the reason for this is I feel it is important to check/preserve business logic that the app,bin,lib,etc dirs are installed without any `[scheduler] install` section.  Hope this is ok.